cmd/messages_text.go

```diff
@@ -98,11 +110,11 @@ func (mt *MessagesText) createMessage(m discord.Message) {
 		mt.createHeader(mt, m, false)
 		mt.createBody(mt, m, false)
 		mt.createFooter(mt, m)
-
-		// Tags with no region ID ([""]) don't start new regions. They can therefore be used to mark the end of a region.
-		fmt.Fprint(mt, `[""]`)
+		
 		fmt.Fprintln(mt)
```

Comment:
Add a default case to the switch block that only displays the name of the author and/or timestamp of the message using the `createHeader` function.

Reply:
added the default case, and also moved the Println outside of the switch case so it applies to every message.